### PR TITLE
Add remaining flower images, update admin for incentives.

### PR DESCRIPTION
**Pull request by eschlange**
* Added missing flower images.
* Updated controller for incentives to include image paths.
* Updated view for incentive creation and editing.

[#92071826]

### CODE REVIEW

File: app/controllers/incentives_controller.rb [hunk 1]
```diff
@@ -62,6 +62,12 @@ def set_incentive
     @incentive = Sunnyside::Incentive.find(params[:id])
   end
 
+  def populate_image_choices
+    @incentive_images = %w(flower1.png flower2.png flower3.png flower4.png
```
Comment (michaelwehrley) [inline app/controllers/incentives_controller.rb]: ruby :golf:  `(1..12).map{|f| "flower#{f}.png"}`

File: app/views/incentives/_form.html.erb [hunk 2]
```diff
@@ -14,6 +14,28 @@
...
+        <div class="row">
+      <% end %>
+
+      <div class="col-md-2">
+        <%= f.radio_button(:image_url,
+                           incentive_image_url) %>
+        <%= image_tag(incentive_image_url, size: "100x100") %>
+      </div>
+
+      <% if index % 6 == 5 || @incentive_images.size == index - 1 %>
```
Comment (michaelwehrley) [inline app/views/incentives/_form.html.erb]: What happens if the above `if index % 6 == 0` is true.  Will the closing div _always_ be displayed.

File: app/controllers/incentives_controller.rb (hunk 1 above)
Comment (eschlange) [inline app/controllers/incentives_controller.rb]: If I was sure that other images would be named using the same format, I would do this. We're supposed to add images of trees and other things in the near future.

Comment (michaelwehrley) [inline app/controllers/incentives_controller.rb]: :sound: good!

File: app/views/incentives/_form.html.erb (hunk 2 above)
Comment (eschlange) [inline app/views/incentives/_form.html.erb]: Yes, it should. Are there any example of it not happening?

Comment (eschlange) [inline app/views/incentives/_form.html.erb]: Updating this to use the auto-wrapping bootstrap row functionality.

**Pull request merged**